src/library/dlgtrackinfo.h

```diff
@@ -121,6 +123,8 @@ class DlgTrackInfo : public QDialog, public Ui::DlgTrackInfo {
     TapFilter m_tapFilter;
     mixxx::Bpm m_lastTapedBpm;
 
+    QMultiHash<QString, QWidget*> m_propertyWidgets;
```

Comment:
> A multi-valued hash is a hash that allows multiple values with the same key.
Is this the case here? 

```suggestion
    QHash<QString, QWidget*> m_propertyWidgets;
```

Reply:
Nope, just dumb copy/pasting.
fixed